### PR TITLE
chore: change lance ref

### PR DESCRIPTION
**Pull request by tychoish**
specifying branch is more clear than hash

### CODE REVIEW

File: crates/sqlbuiltins/Cargo.toml [hunk 1]
```diff
@@ -22,18 +22,17 @@ futures = { workspace = true }
 object_store = { workspace = true }
 thiserror.workspace = true
 tracing = { workspace = true }
-uuid = { version = "1.7.0", features = ["v4", "fast-rng", "macro-diagnostics"] }
+tempfile = { workspace = true }
```
Comment (greyscaled) [inline crates/sqlbuiltins/Cargo.toml:25]: looking at the diff, this seems like it was added/new?

Comment (tychoish) [inline crates/sqlbuiltins/Cargo.toml]: yeah, this is from my branch, will omit

**Comment by scsmithr** [comment]
needs cargo.lock update

**Comment by tychoish** [comment]
on it.

**Pull request merged**